py4DSTEM/visualize/overlay.py

```diff
@@ -688,7 +688,7 @@ def add_scalebar(ax,d):
 
     # Add label
     if label:
-        labeltext = str(length_units)+' '+pixelunits
+        labeltext = f'{np.round(length_units,3)}'+' '+pixelunits
```

Comment:
```suggestion
        labeltext = f'{np.round(length_units,3)} {pixelunits}'
```

Reply:
good change which i somehow did not incorporate, adding now to another PR